packages/camera/camera_android_camerax/android/src/test/java/io/flutter/plugins/camerax/CameraInfoTest.java

```diff
@@ -53,7 +58,47 @@ public void getSensorRotationDegreesTest() {
   }
 
   @Test
-  public void flutterApiCreateTest() {
+  public void getExposureState_retrievesExpectedExposureState() {
+    final CameraInfoHostApiImpl cameraInfoHostApiImpl =
+        new CameraInfoHostApiImpl(mockBinaryMessenger, testInstanceManager);
+    final ExposureState mockExposureState = mock(ExposureState.class);
+    final Long mockCameraInfoIdentifier = 27L;
+    final Long mockExposureStateIdentifier = 47L;
+
+    testInstanceManager.addDartCreatedInstance(mockCameraInfo, mockCameraInfoIdentifier);
+    testInstanceManager.addDartCreatedInstance(mockExposureState, mockExposureStateIdentifier);
+
+    when(mockCameraInfo.getExposureState()).thenReturn(mockExposureState);
+
+    assertEquals(
+        cameraInfoHostApiImpl.getExposureState(mockCameraInfoIdentifier),
+        mockExposureStateIdentifier);
+    verify(mockCameraInfo).getExposureState();
+  }
+
+  @Test
+  @SuppressWarnings("unchecked")
```

Comment:
This has to be used because I mock the generic type `LiveData`, even with the cast.